roles/acme_certificate/tasks/dns-inwx-create.yml

```
@@ -0,0 +1,27 @@
+---
+# Create DNS challenges for DNS provider INWX
+
+- name: Creating challenge DNS entries for {{ ', '.join(acme_certificate_domains + acme_certificate_ips) }} via INWX
+  inwx.collection.dns:
+    state: present
+    domain: "{{ item.0.key | community.dns.get_registrable_domain }}"
+    type: TXT
+    record: "{{ item.0.key | community.dns.remove_registrable_domain }}"
+    value: "{{ item.1 }}"
+    solo: "{{ 'true' if ansible_loop.first else 'false' }}"
+    ttl: 300
+    username: "{{ acme_certificate_inwx_username | default(omit) }}"
+    password: "{{ acme_certificate_inwx_password | default(omit) }}"
+  delegate_to: localhost
+  run_once: true
+  loop_control:
+    extended: yes  
```

```suggestion
    extended: yes
```
Here are the trailing spaces.

I ran `yamllint` manually, it was then much more helpful with its error message:
```
$ yamllint .
./roles/acme_certificate/tasks/dns-inwx-create.yml
  18:18     error    trailing spaces  (trailing-spaces)
```
No idea why in CI it is so ... unhelpful :)